cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSIKafkaSinkTest.java

```diff
@@ -130,13 +130,13 @@ public void testBuildTopicNameDmByServicePath() throws Exception {
     public void testBuildTopicNameDmByServicePathWithSlashServicePath() throws Exception {
         System.out.println(getTestTraceHead("[OrionKafkaSink.buildTopicName]") + "-------- When the "
                 + "root service-path is notified/defaulted and data_model=dm-by-service-path, "
-                + "the Kafka topic name is <service>");
+                + "the Kafka topic name is the concatenatin of <service> and <service-path>");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1133/commits/3ba31a420f9ddda138c790c11f4d8fdc426ba8f4